### PR TITLE
Feature/iosas 853

### PR DESCRIPTION
**Pull request by mmumby**
## Main Updates
- Added links to school application form labels.
- Updated required fields for school application form
- Disabled tabs until form is valid, allowing step through process for filling out the form (Note, the happy path works, ther eis a bug with raido buttons that will be addressed in a followup) 

### Additional Change
- Updated 'status' to 'iosas_reviewstatus' for eoi application payload.

### CODE REVIEW

File: frontend/src/utils/validation.js [hunk 1]
```diff
@@ -1,26 +1,50 @@
-import {DateTimeFormatterBuilder, LocalDate, ResolverStyle} from '@js-joda/core';
-import {groupBy, isPlainObject} from 'lodash';
-
-const datePatternWithSlash = (new DateTimeFormatterBuilder).appendPattern('uuuu/MM/dd').toFormatter(ResolverStyle.STRICT);
+import {
```
Comment (mmumby) [inline frontend/src/utils/validation.js:1]: This file was just formatted

File: frontend/src/utils/institute/formRules.js [hunk 2]
```diff
@@ -105,14 +106,17 @@ const gradeRangeRule = (
   message = 'End grade cannot be before start grade'
 ) => {
   if (startGrade && endGrade) {
+    console.log(
```
Comment (fwpushan) [inline frontend/src/utils/institute/formRules.js]: Is this log required here?

File: frontend/src/utils/validation.js [hunk 3]
```diff
@@ -1,26 +1,50 @@
...
   }
   const penDigits = [];
 
-  for(let i = 0; i < pen.length; i++) {
+  for (let i = 0; i < pen.length; i++) {
     penDigits[i] = parseInt(pen.charAt(i), 10);
   }
-  const S1 = penDigits.slice(0,-1).filter((element,index) => {return index % 2 === 0;}).reduce((a,b) => a+b,0);
-  const A = parseInt(penDigits.slice(0,-1).filter((element,index) => {return index % 2 === 1;}).join(''), 10);
+  const S1 = penDigits
```
Comment (fwpushan) [inline frontend/src/utils/validation.js:21]: Can we use a little meaningful naming here? The SonarQube basic rules set always recommend a variable name of more than three characters.

**Review by fwpushan — CHANGES_REQUESTED** [review]
All good check my comments for some minor feedback.

File: frontend/src/utils/validation.js (hunk 3 above)
Comment (mmumby) [reply frontend/src/utils/validation.js:21]: This file was only changed for formatting - likely all of this code will be deleted as its left over from the cloned app. 
I do agree with variable names, and will ensure that any functions/variables I name will be longer and more descriptive, but I will leave this for now as its likely to be deleted when we eventually get that ticket.

**Review by fwpushan — APPROVED** [review]
All good

**Pull request merged**